blog/_posts/2016-10-03-the-codepen-is-mightier-than-the-sword.md

```diff
@@ -46,11 +46,11 @@ That's exactly what Jekyll's [include](https://jekyllrb.com/docs/templates/) tag
 
 How cool is that? Now whenever I want to embed some code, I just copy that one line, and then change the `slug-hash` and the `height` attributes.
 
-One more neat thing is that thanks to [some cooperation](https://meta.discourse.org/t/embedding-pens-from-codepen/13578) between Discourse and CodePen, this automagically works with [the forum](http://forum.HappyCoding.io). You don't even have to get the embedding HTML and JavaScript. Just paste the url of a pen (like [http://codepen.io/KevinWorkman/pen/zKEkKA](http://codepen.io/KevinWorkman/pen/zKEkKA)) into a forum post, and it will automatically become an embedded code editor. Check it out [here](http://forum.happycoding.io/t/testing-codepen/21/1)!
+One more neat thing is that thanks to [some cooperation](https://meta.discourse.org/t/embedding-pens-from-codepen/13578) between Discourse and CodePen, this automagically works with [the forum](http://forum.HappyCoding.io). You don't even have to get the embedding HTML and JavaScript. Just paste the url of a pen (like [http://codepen.io/KevinWorkman/pen/zKEkKA](http://codepen.io/KevinWorkman/pen/zKEkKA)) into a forum post, and it will automatically become an embedded code editor. Check it out [Happy Coding Forum - Testing-Codepen](http://forum.happycoding.io/t/testing-codepen/21/1)!
```

Comment:
To keep the grammar here correct, can you make this sentence:

`Check it out on the [Happy Coding forum]((http://forum.happycoding.io/t/testing-codepen/21/1)!`